corehq/toggles/__init__.py

```diff
@@ -2708,3 +2708,11 @@ def domain_has_privilege_from_toggle(privilege_slug, domain):
     namespaces=[NAMESPACE_DOMAIN],
     description='The project logo replaces the CommCare logo.',
 )
+
+SUPPORT_GEO_JSON_EXPORT = StaticToggle(
+    slug='support_geo_json_export',
+    label='Support geo json export in Case Exporter',
```

Comment:
Super nitpicky, I know, but we've spelt it "GeoJSON" everywhere else, so I reckon we should spell it like that here.
```suggestion
    label='Support GeoJSON export in Case Exporter',
```